virtcontainers/clh.go

```diff
@@ -426,25 +387,14 @@ func (clh *cloudHypervisor) addDevice(devInfo interface{}, devType deviceType) e
 
 	switch v := devInfo.(type) {
 	case Endpoint:
-		clh.Logger().WithField("function", "addDevice").Infof("Adding Endpoint of type %v", v)
-		clh.cliBuilder.AddNet(CLINet{
-			device: v.Name(),
-			mac:    v.HardwareAddr(),
-		})
+		clh.addNet(v)
 	case types.HybridVSock:
-		clh.Logger().WithFields(log.Fields{
-			"function": "addDevice",
-			"path":     v.UdsPath,
-			"cid":      v.ContextID,
-			"port":     v.Port,
-		}).Info("Adding HybridVSock")
-		clh.cliBuilder.SetVsock(&CLIVsock{
-			cid:        uint32(v.ContextID),
-			socketPath: v.UdsPath,
-			iommu:      false,
-		})
+		clh.addVSock(defaultGuestVSockCID, v.UdsPath)
+	case types.Volume:
+		err = clh.addVolume(v)
 	default:
 		clh.Logger().WithField("function", "addDevice").Warnf("Add device of type %v is not supported.", v)
+		return fmt.Errorf("Not implemented support for %s", v)
```

Comment:
Are we adding any docs with this PR?  I think its important to call out that we don't support block devices, specifically.